dev/package.json

```diff
@@ -1,15 +1,15 @@
 {
     "name": "framer-motion--dev",
-    "version": "7.5.0",
+    "version": "7.5.1-beta.0",
     "private": true,
     "scripts": {
         "dev": "webpack serve --config ./webpack/config.js --hot"
     },
     "dependencies": {
         "@react-three/drei": "^7.27.3",
         "@react-three/fiber": "^8.2.2",
-        "framer-motion": "^7.5.0",
-        "framer-motion-3d": "^7.5.0",
+        "framer-motion": "^7.5.1-beta.0",
+        "framer-motion-3d": "^7.5.1-beta.0",
```

Comment:
should we include beta versions here?

Reply:
this is just cruft really, it'll get updated to 7.5.1 when I publish from main